src/mono/wasm/debugger/BrowserDebugHost/BrowserDebugHost.csproj

```diff
@@ -8,4 +8,6 @@
     <ProjectReference Include="..\BrowserDebugProxy\BrowserDebugProxy.csproj" />
   </ItemGroup>
 
+  <Target Name="GetFilesToPackage" />
```

Comment:
Why not include files to package from here instead of hardcoding paths in the pkgproj?